repair/row_level.cc

```diff
@@ -676,6 +676,28 @@ void flush_rows(schema_ptr s, std::list<repair_row>& rows, lw_shared_ptr<repair_
     }
 }
 
+struct global_reader_permit_meta {
+    std::vector<foreign_ptr<lw_shared_ptr<semaphore_units<>>>> permits{smp::count};
+};
+
+future<global_reader_permit_meta> get_global_reader_permit(repair_service& rs, unsigned shard, bool all_shards) {
+    global_reader_permit_meta meta;
+    co_await rs.container().invoke_on(smp::count -1, [&meta, shard, all_shards] (repair_service& rs) -> future<> {
```

Comment:
What's this? What's special about the last shard?

Reply:
We need to take the semaphore on a single shard. We usually do this on shard zero for a per node operation. Shard zero is already busy enough, so I am using the last shard.